### PR TITLE
VR > Responsive refinements

### PR DESCRIPTION
**Pull request by juanfra**
🎫 https://central.tri.be/issues/126105

### CODE REVIEW

**Comment by bordoni** [comment]
Lets make sure tests are good. 
I changed the parent branch. Lets work on `release/B19.07`, avoiding Huge PRs at the end.

Also Labels, dont forget to add those.

**Comment by juanfra** [comment]
These are pcss changes, the tests failures don't have to do with these.

File: src/resources/postcss/components/_top-bar.pcss [hunk 1]
```diff
@@ -43,12 +43,12 @@
...
+	}
 
-		.tribe-common-h3 {
-			font-size: var(--font-size-6);
-			@media (--viewport-medium) {
-				font-size: var(--font-size-8);
-			}
+	.tribe-events__top-bar-today--title {
+		font-size: var(--font-size-6);
+		@media (--viewport-medium) {
```
Comment (ryanurban) [inline src/resources/postcss/components/_top-bar.pcss]: Are y'all doing any pcss linting?

File: src/views/v2/top-bar/today.php [hunk 2]
```diff
@@ -18,7 +18,7 @@
 		<?php esc_html_e( 'Today', 'the-events-calendar' ); ?>
 	</a>
 
-	<span class="tribe-common-h3 tribe-common-h3--alt">
+	<span class="tribe-common-h3 tribe-common-h3--alt tribe-events__top-bar-today--title">
 		<?php esc_html_e( 'Now', 'the-events-calendar' ); ?> &mdash; <?php echo date( 'F jS, Y', time() ); ?>
```
Comment (ryanurban) [inline src/views/v2/top-bar/today.php]: We should wrap the date in this string in a `<time datetime="..."></time>` for semantic goodness 😃

File: src/resources/postcss/components/_top-bar.pcss (hunk 1 above)
Comment (ryanurban) [reply src/resources/postcss/components/_top-bar.pcss]: @paulmskim

File: src/views/v2/top-bar/today.php (hunk 2 above)
Comment (ryanurban) [reply src/views/v2/top-bar/today.php]: @juanfra

File: src/resources/postcss/components/_top-bar.pcss (hunk 1 above)
Comment (paulmskim) [reply src/resources/postcss/components/_top-bar.pcss]: Not sure, I wasn't too worried about it tbh as we could go back later and lint the heck out of the pcss haha.

**Pull request merged**